client/package.json

```diff
@@ -26,7 +26,8 @@
     "react-router-dom": "^5.1.2",
     "react-virtualized-auto-sizer": "^1.0.2",
     "react-window": "^1.8.5",
-    "uuid": "^8.0.0"
+    "uuid": "^8.0.0",
+    "y18n": "^5.0.5"

```

Comment:
Why are we adding this dependency directly inside `package.json`? `y18n` should be coming to us via some transitive relationship. IMHO, let's try to understand what that relation is and then try fixing the dependency that we actually depend on, instead of having adding a dependency on a new component.

Reply:
on installing the latest version of the y18n package, this dependency got added directly in package.json 
sure, I'll see what the relation is for y18n